lib/get_order_command.rb

```diff
@@ -0,0 +1,21 @@
+require 'order'
+
+class GetOrderCommand
+  attr_reader :response
+
+  def initialize(message)
+    @user_message = message
+  end
+
+  def response?
+    response
+  end
+
+  def run()
+    user_message = @user_message.gsub("order: ", "")
+    user_id_meal_researched = user_message[/(?<=\<@)(\w+)(?=>)/]
```

Comment:
Can you explain this regex in a method or variable? I couldn't quickly digest it 😢